mullvad-types/src/constraints/set.rs

```diff
@@ -0,0 +1,16 @@
+//! A limited variant of Sets.
```

Comment:
Slightly unclear module structure. Why does `Intersection` and `Matches` not both live in `mod.rs`?

Reply:
I'm open to ideas. `Intersection` has to live in either `set.rs` or `constraint.rs` as is, because `constraint.rs` depend on it. If it would live in `mod.rs`, there would be a cyclic dependency :)